packages/installer/upgrade.sh

```diff
@@ -171,23 +171,22 @@ upgrade() {
     # FIXME: Define default /var/tmp as tmpdir_base in default luet config file
     export XDG_RUNTIME_DIR=$temp_upgrade
     export TMPDIR=$temp_upgrade
-
+    args=""
+    if [ -z "$VERIFY" ]; then
+        args="--plugin image-mtree-check"
+    fi
     if [ -n "$CHANNEL_UPGRADES" ] && [ "$CHANNEL_UPGRADES" == true ]; then
-        if [ -z "$VERIFY" ]; then
-          args="--plugin image-mtree-check"
-        fi
         luet install $args --system-target $TARGET --system-engine memory -y $UPGRADE_IMAGE
         luet cleanup
+    elif [ "$DIRECTORY" == true ]; then
+        rsync -axq --exclude='host' --exclude='mnt' --exclude='proc' --exclude='sys' --exclude='dev' --exclude='tmp' ${UPGRADE_IMAGE}/ $TARGET
     else
-        args=""
-        if [ -z "$VERIFY" ]; then
-          args="--plugin image-mtree-check"
```

Comment:
why removing the mtree check from here?

Reply:
it's just a little bit of refactoring, I just moved it up actually to avoid code dups, `args` is just consumed later on need

Reply:
argh, thats true, didnt see the indentation change, nice!